pkg/manifest/manifest_test.go

```diff
@@ -660,23 +676,62 @@ func Test_include(t *testing.T) {
 		},
 		{
 			name:               "incorrect techpreview value is not excluded if techpreview off",
-			includeTechPreview: &falseBol,
+			requiredFeatureSet: utilpointer.String(""),
 			profile:            &defaultClusterProfile,
 			annotations: map[string]interface{}{
 				"include.release.openshift.io/self-managed-high-availability": "true",
 				"release.openshift.io/feature-gate":                           "Other",
 			},
-			expected: fmt.Errorf("unrecognized value release.openshift.io/feature-gate=Other"),
+			expected: fmt.Errorf("unrecognized value \"Other\" in release.openshift.io/feature-gate=Other; known values are: CustomNoUpgrade,Default,IPv6DualStackNoUpgrade,LatencySensitive,TechPreviewNoUpgrade"),
```

Comment:
nit: you might want to pivot `expected` to a regular expression, like [this][1], so folks don't have to come in and bump this when they add or remove feature sets.  On the other hand, folks shouldn't be adding or removing feature sets all that often.

[1]: https://github.com/openshift/library-go/blob/b89722b6e9b5229a53af8906237a50f4b0cbb900/pkg/verify/store/serial/serial_test.go#L71

Reply:
> On the other hand, folks shouldn't be adding or removing feature sets all that often.

This